docs/_includes/ds6/pagination.html

```diff
@@ -0,0 +1,176 @@
+                            <svg aria-hidden="true" focusable="false" width="16" height="16">
+                                <use xlink:href="#svg-icon-arrow-left"></use>
+                            </svg>
+                        </a>
+                        <ol class="pagination__items">
+                            <li class="pagination__item__container">
+                                <a aria-current="page" href="http://www.ebay.com/sch/i.html?_nkw=guitars" class="pagination__item">1</a>
+                            </li>
+                            <li class="pagination__item__container">
+                                <a href="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=2" class="pagination__item">2</a>
+                            </li>
+                            <li class="pagination__item__container">
+                                <a href="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=3" class="pagination__item">3</a>
+                            </li>
+                            <li class="pagination__item__container">
+                                <a href="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=4" class="pagination__item">4</a>
+                            </li>
+                            <li class="pagination__item__container">
+                                <a href="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=5" class="pagination__item">5</a>
+                            </li>
+                            <li class="pagination__item__container">
+                                <a href="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=6" class="pagination__item">6</a>
+                            </li>
+                            <li class="pagination__item__container">
+                                <a href="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=7" class="pagination__item">7</a>
+                            </li>
+                            <li class="pagination__item__container">
+                                <a href="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=8" class="pagination__item">8</a>
+                            </li>
+                            <li class="pagination__item__container">
+                                <a href="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=9" class="pagination__item">9</a>
+                            </li>
+                        </ol>
+                        <a aria-label="Next Page" class="pagination__next" href="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=2">
+                            <svg aria-hidden="true" focusable="false" width="16" height="16">
+                                <use xlink:href="#svg-icon-arrow-right"></use>
+                            </svg>
+                        </a>
+                    </nav>
+                </div>
+            </div>
+            {% highlight html %}
+            <nav class="pagination" aria-labelledby="pagination-heading" role="navigation">
```

Comment:
Please fix indentation of example code:

![screen shot 2018-04-06 at 10 26 01 am](https://user-images.githubusercontent.com/38065/38434953-10e663a2-3985-11e8-8640-81f4beffb1c0.png)